.gitignore

```diff
@@ -4,3 +4,8 @@ node_modules/
 .venv
 .mypy_cache
 __pycache__
+.vs/
+bin/
+obj/
+Debug/
+Release/
```

Comment:
Can you move these to a separate `.gitignore` file in the directory 05_wtelegramclient?